kubernetes/linux/setup.sh

```diff
@@ -40,10 +41,10 @@ sudo apt-get install jq=1.5+dfsg-2 -y
 #used to setcaps for ruby process to read /proc/env
 sudo apt-get install libcap2-bin -y
 
-wget https://dl.influxdata.com/telegraf/releases/telegraf-1.22.2_linux_$ARCH.tar.gz
-tar -zxvf telegraf-1.22.2_linux_$ARCH.tar.gz
+wget https://dl.influxdata.com/telegraf/releases/telegraf-1.23.2_linux_$ARCH.tar.gz
+tar -zxvf telegraf-1.23.2_linux_$ARCH.tar.gz
```

Comment:
did we test with this version to make sure no regressions because of the update?

Reply:
tests are underway

Reply:
![image](https://user-images.githubusercontent.com/10297394/180308564-d9677cd3-03e2-4121-8c73-c303198cd39a.png)

Reply:
![image](https://user-images.githubusercontent.com/10297394/180308643-e9919846-53b5-4f4d-99e2-bd397643a345.png)

Reply:
the pod logs do not show any telegraf errors as well